alphadia/outputtransform.py

```diff
@@ -402,9 +400,9 @@ def build_transfer_model(self):
         tune_mgr = FinetuneManager(
             device=device, settings=self.config["transfer_learning"]
         )
-        stats = tune_mgr.finetune_rt(transfer_lib.precursor_df)
-        stats = tune_mgr.finetune_charge(transfer_lib.precursor_df)
-        stats = tune_mgr.finetune_ms2(
+        stats = tune_mgr.finetune_rt(transfer_lib.precursor_df)  # noqa: F841  # TODO check for potential bug then remove this line
```

Comment:
These lines have to stay. Kinda weird, that ruff want's to remove the function calls.

Reply:
ruff wants to remove the unused variables, and is not aware of the called methods altering some internal state in `tune_mgr` .. 
ruff is happy if the variable name is changed to "_", i.e.
```_ = tune_mgr. ...```
or if the variable assignment is removed .. I'll do the latter ..

Reply:
=> https://github.com/MannLabs/alphadia/pull/236